### ISSUE TITLE
Allow `make test` to pass -v flag

### ISSUE THREAD

**Issue by surajssd** [redhat-developer/opencompose#75]
$subject

This helps in knowing the errors where tests are failing and also see the logs of tests the way they are run.

**Comment by tnozicka** [comment]
Possible, but if the test fails you can always run it by `go test` with what ever parameters you want, right?

**Comment by surajssd** [comment]
yes, that is possible but having a flag will help and I think in travis the tests should run in verbose mode anyway.

**Comment by kadel** [comment]
> yes, that is possible but having a flag will help and I think in travis the tests should run in verbose mode anyway.

I'm not sure about that to be honest. I know that Kompose uses `-v` by default for test. But I've never found it useful, and in most cases just annoying. 

If I'm not mistaken if test fails, it  shows output for that test as it was run with `-v`
